evalai/challenges.py

```diff
@@ -234,4 +235,48 @@ def submit(ctx, file):
     make_submission(ctx.challenge_id, ctx.phase_id, file, submission_metadata)
 
 
+@phase.command(context_settings={"ignore_unknown_options": True})
+@click.pass_obj
+@click.option("--file", type=click.STRING, required=True, help="File path to the submission file")
+def upload_submission(ctx, file_name):
+    """
+    For uploading large submission files to S3
+    Invoked by runing 'evalai challenge CHALLENGE phase PHASE upload_submission FILE'
+
+    Arguments:
+        ctx (class click.Context) --  The context object which holds state of the invocation
+        file_name (str) -- the path of the file to be uploaded
+    """
+    submission_metadata = {}
+    if click.confirm("Do you want to include the Submission Details?"):
+        submission_metadata["method_name"] = click.prompt(
+            style("Method Name", fg="yellow"), type=str, default=""
+        )
+        submission_metadata["method_description"] = click.prompt(
+            style("Method Description", fg="yellow"), type=str, default=""
+        )
+        submission_metadata["project_url"] = click.prompt(
+            style("Project URL", fg="yellow"), type=str, default=""
+        )
+        submission_metadata["publication_url"] = click.prompt(
+            style("Publication URL", fg="yellow"), type=str, default=""
+        )
+    upload_submission_file_with_presigned_url(ctx.phase_id, file_name, submission_metadata)
+
+
+@phase.command(context_settings={"ignore_unknown_options": True})
+@click.pass_obj
+@click.option("--file", type=click.STRING, required=True, help="File path to the annotations file")
+def upload_annotation(ctx, file_name):
+    """
+    For uploading large test annotations for a challenge phase to S3
+    Invoked by runing 'evalai challenge CHALLENGE phase PHASE upload_annotations FILE'
+
+    Arguments:
+        ctx (class click.Context) --  The context object which holds state of the invocation
+        file_name (str) -- the path of the file to be uploaded
```

Comment:
is this file name or file path? if param is path can we rename it to `file_path` if not can you change the description?

Reply:
Changing this to `--file` for now.
This is simple and the meaning seems straightforward to the host. I have however changed the method signature to ensure code clarity (i.e, file_name -> file_path)